.github/workflows/test.yaml

```diff
@@ -82,6 +82,24 @@ jobs:
         with:
           aqua_version: v1.25.0
 
+      - name: Restore test images from cache
+        uses: actions/cache@v4
+        id: restore-test-images
+        with:
+          path: integration/testdata/fixtures/images
+          key: cache-test-images-2024-09-10 # trivy-test-images last update date
+
+      - name: Download test images
+        if: steps.restore-test-images.outputs.cache-hit != 'true'
```

Comment:
It pulls only images that don't exist locally. We should not skip it even if a cache hits.
https://github.com/aquasecurity/trivy/blob/0767cb84430cf0214b2211492f046b257a314347/magefiles/fixture.go#L29-L32

Reply:
Oh, forget about it. Matching `restores-key` doesn't return true.

>Note cache-hit will only be set to true when a cache hit occurs for the exact key match. For a partial key match via restore-keys or a cache miss, it will be set to false.